06_functions/main.go

```diff
@@ -74,42 +73,67 @@ Note: functions can be defined in any order in the file.
 
 func main() {
 	sayHello("John")
+	fmt.Println(diff(1, 9))
+	fmt.Println(diff(9, 1))
+	fmt.Println(diff(4, -10))
+	printInput("안녕", 3)
+	printInput("go", 7)
+	printInput("ça va", 4)
+}
+
+// For each exercise, call the functions a couple of times with different
+// values and print the results.
 
-	// For each exercise, call the functions a couple of times with different
-	// values and print the results.
+// TODO: Write a function that takes two integers as parameters and returns
+// the difference of the two integers.
+func diff(a int, b int) int {
+	var sum int = b - a
+	return sum
+}
 
-	// TODO: Write a function that takes two integers as parameters and returns
-	// the difference of the two integers.
+// TODO: Write a function that takes a string and an integer as parameters.
+// The function should print the string as many times as the integer.
+func printInput(c string, d int) {
```

Comment:
Using good naming for parameters is crucial.
It's a bit subjective but using 1 letter is usually frown upon.
How about:
```suggestion
func printInput(str string, count int) {
```

Reply:
That sounds good, thanks!